app/src/main/kotlin/com/intive/picover/profile/model/ProfileUpdateResult.kt

```diff
@@ -0,0 +1,7 @@
+package com.intive.picover.profile.model
+
+import java.io.Serializable
+
+data class ProfileUpdateResult(
```

Comment:
Is this wrapper mandatory?

Reply:
It might be useful in handling results in voyager, because there's no out of the box solution for that yet, although there are some open PRs and discussions about that, I would revisit that during the migration